client/lcd/root.go

```diff
@@ -45,20 +46,22 @@ func NewRestServer(cdc *codec.Codec) *RestServer {
 		Mux:    r,
 		CliCtx: cliCtx,
 		Cdc:    cdc,
-
-		log: logger,
+		log:    logger,
 	}
 }
 
 // Start starts the rest server
-func (rs *RestServer) Start(listenAddr string, maxOpen int) (err error) {
+func (rs *RestServer) Start(listenAddr string, maxOpen int, readTimeout, writeTimeout uint) (err error) {
 	server.TrapSignal(func() {
 		err := rs.listener.Close()
 		rs.log.Error("error closing listener", "err", err)
 	})
 
-	cfg := rpcserver.DefaultConfig()
```

Comment:
Why are you dropping `DefaultConfig()`? Couldn't we override only the values that we want to override and fallback to the defaults?

Reply:
Because there is no longer a need for `DefaultConfig()`. `Config` is defined by three values which we each set manually. ie. the call would be redundant.

Reply:
I see - fair enough